attesters/csv/collect_evidence.c

```diff
@@ -178,8 +184,7 @@ typedef struct {
  * 	0: success
  * 	otherwise error
  */
-static int collect_attestation_evidence(const uint8_t *hash, uint32_t hash_len,
-					csv_attestation_evidence_t *evidence)
+static int collect_attestation_evidence(const uint8_t *hash, uint32_t hash_len, csv_evidence *evidence)
```

Comment:
please use `clang-format` to format your code.

Reply:
@haosanzi , done